testapps/testapp/src/main/java/com/microsoft/identity/client/testapp/nativeauth/SignUpPasswordFragment.kt

```diff
@@ -0,0 +1,132 @@
+package com.microsoft.identity.client.testapp.nativeauth
```

Comment:
Let's add License to all these new files

Reply:
Done, and for all other .kt files